### ISSUE TITLE
Adding mod when required research already complete fails to add the item to the game

### ISSUE THREAD

**Issue by MrDrummer** [theRustyKnife/CircuitNetworkSwitch#2]
I've added this mod since researching the required tech, and the switch is not available.

I think you need to do a simple check on first load if the required research is already completed, and if so, unlock the item.

EDIT:
Screenshot of research:
![image](https://cloud.githubusercontent.com/assets/10013572/24204818/8059b2a6-0f11-11e7-95d9-2558eb90378f.png)
Screenshot of Logistics tab:
![image](https://cloud.githubusercontent.com/assets/10013572/24204854/a39a4ff0-0f11-11e7-8106-870c3bd6bdf4.png)

**Comment by theRustyKnife** [comment]
You are completely correct, I'll fix this once I'm done with updating Crafting Combinator.
Cheers TRK
